### DISCUSSION TITLE
What is the ideal way to write to a single db concurrently?

### DISCUSSION THREAD

**Discussion by ajitpeter** [fnc12/sqlite_orm#780] [General]
I am trying to write to a database from two different instances(executable) and for smaller number of transactions it seems to  be okay. It crashes when trying to write large number of entries into the database simultaneously. Could somebody point me to the documentation or suggest an alternative please?

**Comment by trueqbit** [comment]
Concurrent database access is really a SQLite question. As to why your program crashes would need be investigated.
